### PR TITLE
Clean add this element add this wrapper

### PR DESCRIPTION
**Pull request by ChuChuNaKu**

### CODE REVIEW

**Comment by nerdstein** [comment]
@ChuChuNaKu - which issue are you referencing? please provide with all PRs

File: src/Element/AddThisBasicToolbox.php [hunk 1]
```diff
@@ -32,18 +34,18 @@ public function getInfo() {
...
-      '#attributes' => array(
-        'class' => array(
-          'addthis_toolbox',
-          'addthis_default_style',
-         // ($settings['buttons_size'] == AddThis::CSS_32x32 ? AddThis::CSS_32x32 : NULL),
-          //$settings['extra_css'],
-        ),
-      ),
-    );
+////   $element['addthis_basic_toolbox'] = array(
```
Comment (nerdstein) [inline src/Element/AddThisBasicToolbox.php]: Is there a reason why this is commented out? Plan to use later? If not, remove

File: templates/addthis-basic-toolbox.html.twig [hunk 2]
```diff
@@ -7,4 +7,20 @@
  *
  */
 #}
-{{ element }}
+
+<div class="addthis_toolbox addthis_default_style {{ element['#extra_classes'] }} {{ element['#size'] }} ">
+{#{{ element }}#}
```
Comment (nerdstein) [inline templates/addthis-basic-toolbox.html.twig]: remove?

**Comment by nerdstein** [comment]
Reviewed and really impressed with how much cleanup occured in this PR, great job. I'll review with more context when I get the issue

**Comment by ChuChuNaKu** [comment]
@nerdstein sorry about that. This is for issue number https://github.com/d8-contrib-modules/addthis/issues/16. I had LOTS of help!

File: src/Plugin/Field/FieldFormatter/AddThisBasicToolboxFormatter.php [hunk 3]
```diff
@@ -53,13 +53,12 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
...
   public function viewElements(FieldItemListInterface $items) {
-    $settings = $this->getSettings();
-
-    $add_this_service = \Drupal::service('addthis.addthis');
-    $markup = $add_this_service->getBasicToolboxMarkup($settings);
 
     return array(
-      '#markup' => $markup
+      '#type' => 'addthis_basic_toolbox',
+      '#size' => $config['basic_toolbox']['buttons_size'],
```
Comment (doylejd) [inline src/Plugin/Field/FieldFormatter/AddThisBasicToolboxFormatter.php]: $config does not exist in the context of this function. Blocks use $this->configuration while Fields use $this->getSettings(). You will need to adjust these for the fields.

File: src/Plugin/Field/FieldFormatter/AddThisBasicButtonFormatter.php [hunk 4]
```diff
@@ -52,11 +52,11 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
...
    */
   public function viewElements(FieldItemListInterface $items) {
-    $settings = $this->getSettings();
-    $add_this_service = \Drupal::service('addthis.addthis');
-    $markup = $add_this_service->getBasicButtonMarkup($settings);
+
     return array(
-      '#markup' => $markup
+      '#type' => 'addthis_basic_button',
+      '#size' => $config['basic_button']['button_size'],
```
Comment (doylejd) [inline src/Plugin/Field/FieldFormatter/AddThisBasicButtonFormatter.php]: $config does not exist in the context of this function. Blocks use $this->configuration while Fields use $this->getSettings(). You will need to adjust these for the fields.

**Comment by ChuChuNaKu** [comment]
ok, @nerdstein I got rid of the commented code. @doylejd I reworked the fields.

**Comment by doylejd** [comment]
@ChuChuNaKu Going to merge this so Jason can get his stuff working, but I am pretty sure the config arrays are wrong in the field formatters. Lets get a new PR open for that.

**Pull request merged**